manifests/plugin/remote_execution/cockpit.pp

```diff
@@ -0,0 +1,52 @@
+# Installs remote execution cockpit plugin
+class foreman::plugin::remote_execution::cockpit {
+  require foreman::plugin::remote_execution
+
+  $config_directory = '/etc/foreman/cockpit'
+  $foreman_url = $foreman::foreman_url
+  $cockpit_path = '/webcon'
+  $cockpit_host = '127.0.0.1'
+  $cockpit_port = 9999
+  $cockpit_config = {
+    'foreman_url' => $foreman_url,
+    'ssl_ca_file' => $foreman::client_ssl_ca,
```

Comment:
I have some concerns about this since it's also used to verify the CA on `$foreman_url`. That's not guaranteed to be the same CA.